css/layout.css

```diff
@@ -286,3 +288,154 @@ summary {
 .form .button {
   padding: 20px 30px;
 }
+
+@media screen and (min-width: 992px) {
+  .main-nav img {
+    display: none;
+  }
+
+  .ul-menu {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .main-nav {
+    width: 35%;
+  }
+
+  .header {
+    padding: 40px 120px;
+  }
+
+  .intro .wrapper p {
+    max-width: 750px;
+  }
+
+  .intro .wrapper {
+    padding-top: 120px;
+    min-height: 90vh;
+  }
+
+  .intro {
+    display: flex;
+    justify-content: center;
+  }
+
+  .social {
+    width: 20%;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .card {
+    flex-direction: row;
+    max-width: 100%;
+    gap: 20px;
+    padding: 20px;
+  }
+
+  .card > * {
+    width: 50%;
+  }
+
+  .works .wrapper {
+    max-width: 100%;
+  }
+
+  button {
+    margin-left: 0;
+  }
+
+  .content {
+    padding-top: 30px;
+  }
+
+  .row-reverse {
+    flex-direction: row-reverse;
+  }
+}
+
+@media screen and (min-width: 992px) {

```

Comment:
- [ ] Do not repeat the same breaking point over and over again so that your code looks DRY and professional. Consider moving these styles to one specified desktop breaking point okay 😃